source/documentation/data_warehouse_guide/topics/Overview_of_Configuring_Data_Warehouse.adoc

```diff
@@ -1,5 +1,5 @@
 [[Overview_of_Configuring_Data_Warehouse]]
-=== Overview of Configuring Data Warehouse
+== Overview of Configuring Data Warehouse
```

Comment:
I suggest instead:
`= Overview of Configuring Data Warehouse`

And in doc-Data_Warehouse_Guide/chap-Installing_and_Configuring_Data_Warehouse.adoc:

`include::topics/Overview_of_Configuring_Data_Warehouse.adoc[leveloffset=+1]`

Reply:
Done